src/FillArrays.jl

```diff
@@ -159,11 +159,11 @@ function getindex(F::Fill, kj::Vararg{AbstractVector{II},N}) where {II<:Integer,
 end
 
 function getindex(A::Fill, kr::AbstractVector{Bool})
-   length(A) == length(kr) || throw(DimensionMismatch())
+   length(A) == length(kr) || throw(DimensionMismatch("lengths must match"))
```

Comment:
This is the message that looks wrong to me. E.g.
```julia
Fill(5,10)[trues(9)]
```

Reply:
Oh, this was a foolish attempt on my part to try to be helpful by sticking in a String in another location after you pointed out that I was missing a String in the code I had added.  I have reverted this back to the original string-less version because it will be more efficient for you to update it the way you want instead of me trying to guess.  I should not have mixed unrelated changes to other files in this PR...